docs/reference/ois/next/processing.md

```diff
@@ -77,14 +93,84 @@ post-processing.
 
 :::
 
+### v2
+
+The processing snippet receives parameters submitted as part of a template or
+on-chain request.
+
+Airnode executes snippets for `preProcessingSpecificationV2` and
+`postProcessingSpecificationV2` during its run cycle. The following describes
+the work flow Airnode uses:
+
+1. Run `preProcessingSpecificationV2`
+2. Airnode calls requested OIS endpoint
+3. Run `postProcessingSpecificationV2`
+4. Airnode encodes the response values defined by reservedParameters
+
+The processing schema is the same for both
+[`preProcessingSpecificationV2`](/reference/ois/next/specification.md#_5-11-preprocessingspecificationv2)
+and
+[`postProcessingSpecificationV2`](/reference/ois/next/specification.md#_5-12-postprocessingspecificationv2).
+Snippets for both specifications follow this schema:
+
+- `environment` - Currently only possible value is `Node`. This options
+  interprets the code as JavaScript and execute in Node.js. The function can be
+  also asynchronous (async/await is supported as well). The processing
+  implementation will wait for the function to resolve.
+- `value` - The processing code written as a string.
+- `timeoutMs` - The maximum timeout that this snippet can run. In case the
+  timeout is exceeded an error is thrown.
+
+Try the [Post processing](/guides/airnode/post-processing/) guide to further
+understand pre/post processing.
+
+#### Input and Output
+
+The processing snippet is a function which receives payload as an argument. The
+return value of the function is treated as a processing result. Apart from the
+payload argument, you can use most Node.js built-in modules.
+
+The payload argument for pre-processing is an object with the following
+properties:
```

Comment:
"propert**ies**" plural, but there is only `endpointParameters`?

Reply:
I didn't mind for two reasons:
- It's future proof
- It's not a "cohesive" text. The sentence says that the object has certain properties and follows up with a list of them. The list has just a single item.

That said, youre suggestion is more accurate so I'll use that.